metrics/report/report_dockerfile/pdf.Rmd

```diff
@@ -0,0 +1,23 @@
+---
+# Copyright (c) 2018-2019 Intel Corporation
```

Comment:
this is a new file, so I think only 2019 is ok

Reply:
those snippets were cut out of the top of the `metrics_report.Rmd` file, and separated out - so nominally they are derivatives - they copy included the header, so I updated it.